operator/static/jquery.js

```diff
@@ -64,39 +64,51 @@ $(document).ready(function() {
             alert('Select a mission.');
             return
         }
-
     
-        simulatorIsActive 
-        ? 
-        $.ajax('simulator/exit', {
-            data: JSON.stringify({access_token: accessToken}),
-            contentType: 'application/json',
-            type: 'POST',
-            success: res => {
-                simulatorIsActive = false;
-                logs.push(res.response);
-                $('#launch-exit').removeClass('btn-danger').addClass('btn-primary').text('LAUNCH simulator') 
-                $('.log-window').append(`<p>${res.response}</p>`);
-            },
-            error: res => {
-                alert(res.responseJSON.error);
-            }
-        }) 
-        :
-        $.ajax('simulator/launch', {
-            data: JSON.stringify({id: selectedTeam, mission: selectedMission, access_token: accessToken}),
-            contentType: 'application/json',
-            type: 'POST',
-            success: res => {
-                simulatorIsActive = true;
-                logs.push(res.response);
-                $('#launch-exit').removeClass('btn-primary').addClass('btn-danger').text('EXIT simulator'); 
-                $('.log-window').append(`<p>${res.response}</p>`);
-            },
-            error: res => {
-                alert(res.responseJSON.error);
-            }
-        });
+        const selectedTrack = $("input:radio[name ='track-select']:checked").val();
+        if (selectedTrack === undefined) {
+            alert('Select a track.');
+            return
+        }
+
+        $('#launch-exit').prop('disabled', true);
```

Comment:
disable the button after it is pressed to ensure people don't accidently press it again while it is launching/stopping the simulator.